generated/nidaqmx/system/device.py

```diff
@@ -1161,6 +1174,27 @@ def unreserve_network_device(self):
 
     # endregion
 
+    def get_ext_cal_last_date_and_time(device_name):
```

Comment:
In my opinion, these should be properties, not get methods. 

They are properties in LabVIEW:
https://www.ni.com/docs/en-US/bundle/ni-daqmx-properties/page/daqmxprop/daqmxcalibrationinfo.html

and .NET:
```
class Device
...
        property DateTime SelfCalibrationDateTime { DateTime get(); }
        property DateTime ExternalCalibrationDate { DateTime get(); }
```

```suggestion
    @property
    def ext_cal_last_date_and_time(device_name):
```

Reply:
The get methods are converted to properties.